README.md

```diff
@@ -26,42 +36,53 @@ The component that is generated from the web app is the event website. Several s
+ 
+ **7. [WebdriverJs](https://www.npmjs.com/package/selenium-webdriver)** - *Official Javascript Implementation of Selenium for testing generated event websites*
 
 #### Components of the Generator
 
-1. Webform
+**1. Webform**
+
+ > **The source of the webform can be found [here](/src/www).**
 
- The source of the webform can be found [here](/src/www). It consists of -
+**It consists of :-**
 
- - [index.html](/src/www/index.html) - The webform page
- - [form.js](/src/www/js/form.js) - The script that uploads the zips, and starts the generator process
+ - **[index.html](/src/www/index.html)** - *The webform page*
+ - **[form.js](/src/www/js/form.js)** - *The script that uploads the zips, and starts the generator process*
 
-2. Generator
+**2. Generator**
 
- The generator runs on a ExpressJS server, using this main [app script](/src/app.js).
+ > **The generator runs on a ExpressJS server, using this main [app script](/src/app.js).**
 
-3. Scripts
+**3. Scripts**
 
- - [generator.js](/src/backend/generator.js) - Does the main generation tasks, and controls other scripts
- - [fold.js](/src/backend/fold.js) - Groups data of sessions, speakers, tracks etc from JSONs
- - [dist.js](/src/backend/dist.js) - Creates folders, cleans folders, unzips/zips packages
- - [ftpdeploy.js](/src/backend/ftpdeploy.js) - Deploys finished website to organiser's server (optional)
- - [mailer.js](/src/backend/mailer.js) - Sends mail to organiser notifying of successful creation
- - [buildlogger.js](/src/backend/buildlogger.js) - Displays build logs while generating webapp
- - [deploy.js](/src/backend/deploy.js) - Automatically deploys generated event sites to user's github account
- - [gulpfile.js](/src/backend/gulpfile.js) - Creates minified js and css for generated webapp
+ - **[generator.js](/src/backend/generator.js)** - *Does the main generation tasks, and controls other scripts*
+ - **[fold.js](/src/backend/fold.js)** - *Groups data of sessions, speakers, tracks etc from JSONs*
+ - **[dist.js](/src/backend/dist.js)** - *Creates folders, cleans folders, unzips/zips packages*
+ - **[ftpdeploy.js](/src/backend/ftpdeploy.js)** - *Deploys finished website to organiser's server (optional)*
+ - **[mailer.js](/src/backend/mailer.js)** - *Sends mail to organiser notifying of successful creation*
+ - **[buildlogger.js](/src/backend/buildlogger.js)** - *Displays build logs while generating webapp*
+ - **[deploy.js](/src/backend/deploy.js)** - *Automatically deploys generated event sites to user's github account*
+ - **[gulpfile.js](/src/backend/gulpfile.js)** - *Creates minified js and css for generated webapp*
 
-4. Templates
+**4. Templates**
 
- The HTML pages of the generated website are created using Handelbars templates. You can find all the templates [here](/src/backend/templates) -
+ > The HTML pages of the generated website are created using Handelbars templates. 
```

Comment:
Lets keep this simple, and remove `> `

Reply:
Done, just removed `>` because that was not **bold**